supported_platforms.md

```diff
@@ -0,0 +1,57 @@
+# Microsoft Azure IoT SDKs for .NET
+
+This SDK is tested nightly on a mix of .NET implementations on both Windows 10 and on Ubuntu 1604. For additional details
+for each tested platform, see the respective sections below. 
+
+## Supported .NET versions
+
+The NuGet packages provide support for the following .NET versions:
+- .NET Standard 2.1
+- .NET Standard 2.0
+- .NET Framework 4.7.2 (IoT Hub SDKs only)
+- .NET Framework 4.5.1 (IoT Hub SDKs only)
+
+This SDK _may_ work with other versions of .NET, but there are no guarantees that they will _always_ work for those 
+versions nor are there guarantees that we will fix bugs that are only present on those versions.
+
+Note that applications will resolve the dll that is being referenced based on the framework precedence rules. This means that .NET Framework targeting applications will look for the closet .NET Framework dll. In the absence of that, it will pick up the closest .NET Standard dll. Similarly for netcoreapp applications will look for the closest netcoreapp dll and in the absence of one will pick the closest .NET Standard dll. Since we publish the above list of dlls, you should target the appropriate net target to ensure you get the desired .net API coverage
```

Comment:
nit: uppercase .NET (in the last statement)

Reply:
```suggestion
Note that applications will resolve the dll that is being referenced based on the framework precedence rules. This means that .NET Framework targeting applications will look for the closet .NET Framework dll. In the absence of that, it will pick up the closest .NET Standard dll. Similarly for netcoreapp applications will look for the closest netcoreapp dll and in the absence of one will pick the closest .NET Standard dll. Since we publish the above list of dlls, you should target the appropriate net target to ensure you get the desired .NET API coverage
```